### PR TITLE
Dashboard editor fixes

### PR DESCRIPTION
**Pull request by sergeysenja1992**

### CODE REVIEW

File: packages/repositories/src/http-client.rest.ts [hunk 1]
```diff
@@ -12,6 +12,7 @@ export interface XmRepositoryConfig {
 export class HttpClientRest<T extends IId = unknown, Extra extends Pageable = Pageable> implements IEntityCollectionPageable<T, Extra> {
 
     public readonly loading$: BehaviorSubject<boolean> = new BehaviorSubject<boolean>(false);
+    private loadingCount = 0; // counter of loading that in progress
```
Comment (Kaskyi) [inline packages/repositories/src/http-client.rest.ts:15]: /** counter of loading that in progress */

https://tsdoc.org/

File: packages/administration/dashboards-config/components/widget-edit/widget-edit.component.ts [hunk 2]
```diff
@@ -143,6 +148,11 @@ export class WidgetEditComponent implements OnChanges {
     }
 
     public onSave(): void {
+        if (!this.valid) {
+            console.warn("Widget isn't valid")
```
Comment (Kaskyi) [inline packages/administration/dashboards-config/components/widget-edit/widget-edit.component.ts:152]: you do not need this warn
you have:
[disabled]="!form.valid || disabled || !valid"

**Review by Kaskyi — APPROVED** [review]

**Review by Bozonych — APPROVED** [review]

File: packages/repositories/src/http-client.rest.ts (hunk 1 above)
Comment (sergeysenja1992) [reply packages/repositories/src/http-client.rest.ts:15]: this is not documentation, this is a comment for a non-trivial part of the code

Comment (Kaskyi) [reply packages/repositories/src/http-client.rest.ts:15]: imho it is alredy selt describbed

Comment (sergeysenja1992) [reply packages/repositories/src/http-client.rest.ts:15]: if everything was obvious here, there would be no bug =))

**Pull request merged**